data/json/mutations.json

```diff
@@ -487,7 +490,12 @@
         "points" : 1,
         "description" : "You have no qualms about bending the truth, and have practically no tells.  Telling lies and otherwise bluffing will be much easier for you.",
         "starting_trait" : true,
-        "cancels" : ["TRUTHTELLER"]
+        "cancels" : ["TRUTHTELLER"],
+        "social_modifiers" : {
+            "//" : "The tail acts as a tell, preventing this trait from working",
```

Comment:
That is very specific and weird.
Would be much better without this single special case and the whole `blocker_mutations` mechanic that needs to be maintained for this tiny case.

Reply:
I kind of agree, I was just replicating the logic was already in the hardcoded version.  I can drop it and simplify if you think it makes sense.  On the other hand, the system might come in handy if social modifiers were applied to more mutations.

Thoughts?  I'm okay with going either way on this one

Reply:
It looks like someone just added an `else` by accident. Most such weird interactions have explicit comments.

Reply:
Fair enough.  I thought I remembered it being an intentional interaction, but my memory is probably just failing me.  Either way, I'll cut it out for now

Reply:
I'm also fine with just removing this.